db/system_distributed_keyspace.cc

```diff
@@ -394,5 +394,27 @@ system_distributed_keyspace::cdc_get_versioned_streams(context ctx) {
     });
 }
 
+future<db_clock::time_point>
+system_distributed_keyspace::get_next_stream_timestamp(context ctx, db_clock::time_point low) {
+    // I love that you cannot order by pk.
```

Comment:
As much as I enjoy the humor. I don't think it's a good idea to leave it in the code.

Reply:
This is not humour. I am dead serious.